app/actions/service_credential_binding_key_create.rb

```diff
@@ -4,19 +4,35 @@ class ServiceCredentialBindingKeyCreate
       class UnprocessableCreate < StandardError
       end
 
-      def precursor(service_instance, volume_mount_services_enabled: false)
+      def precursor(service_instance, name)
+        validate!(service_instance)
+
+        binding_details = {
+          service_instance: service_instance,
+          name: name,
+          credentials: {}
+        }
+
+        ServiceKey.db.transaction do
```

Comment:
should we do this in the model to keep it closer to what we do for credential bindings?

Reply:
We'll refactor this later